### ISSUE TITLE
configure: Catch2 support shows no status

### ISSUE THREAD

**Issue by joachimmetz** [sleuthkit/sleuthkit#3039]
Catch2 support in the configure output is missing a status indicator

```
Building:
   afflib support:                        no
   libaff4 support:                       no
   libbfio support:                       yes
   libewf support:                        no
   libqcow support:                       yes
   libvhdi support:                       no
   libvmdk support:                       no
   libvslvm support:                      no

   openssl support:                       no
   zlib support:                          yes

Features:
   Catch2 support:                        
   Java/JNI support:                      no
   Multithreading:                        yes
```

**Comment by joachimmetz** [comment]
There appears to be no configure test that sets with_catch2 ?

**Comment by uckelman-sf** [comment]
@simsong Closed by #3040.
